lisa/operating_system.py

```diff
@@ -388,9 +388,10 @@ def install_packages(
         signed: bool = False,
         timeout: int = 1200,
         extra_args: Optional[List[str]] = None,
+        sudo: bool = True,
```

Comment:
did you find any package installation command are blocked? Please provide examples here. We can think about if it can be detected in code, instead of depending on configuration.